lte/gateway/python/magma/subscriberdb/main.py

```diff
@@ -66,9 +64,7 @@ def main():
             service_stub=SubscriberDBCloudStub,
             max_client_reuse=60,
         )
-        sync_interval = _randomize_sync_interval(
-            service.config.get('subscriberdb_sync_interval'),
-        )
+        sync_interval = service.mconfig.sync_interval
```

Comment:
@hcgatewood Is this completely safe? I think it is because a `sync_interval` has been added to the default `gateway.mconfig` and we have made sure that the orc8r sends one too.

Reply:
Seems safe to me. Will defer to @ardzoht. Also, you can probably test this yourself with local AGW setup if you want to, but may not be necessary

Reply:
I tested it with an AGW connected to the orc8r. Will test with a headless one too.

Reply:
Adding this to the default gateway.mconfig and validating this field is always generated by orc8r should be enough,  you can always do `if service.mconfig.sync_interval` to validate existence but it should be good as it is.